packages/react/src/components/Tabs/tabPanel.test.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useRef } from 'react';
+import { render, screen, waitFor } from '@testing-library/react';
+import { TabPanel } from './';
+
+test('should render tab panel with two paragraphs correctly', async () => {
+  const TabPanelwithRef = () => {
+    const ref = useRef(null);
+    return (
+      <div ref={ref}>
```

Comment:
To be perfectly honest, I don't really remember why we have this wrapped in a ref, so we may be able to omit the wrapping `div` and just render `TabPanel` directly for this and the other examples.